src/app/layout/components/app-layout/app-layout.component.ts

```diff
@@ -37,10 +41,17 @@ export class AppLayoutComponent implements OnInit, OnDestroy {
     this.matcher.addEventListener('change', (event) => {
       this.isSmallDeviceListener(event)
     })
+
+    this.subscriptions.add(
+      this.profileService.getUnapprovedUser().subscribe((response: any) => {
+        this.unapprovedUser = response === true
```

Comment:
why not just this.unapprovedUser = response  ?

Reply:
Yep, no reason for that to test